examples/medplum-valueset-selector/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,209 @@
+import { Title, Group, TextInput, Box, Textarea, Button, Alert, Grid } from '@mantine/core';
+import { CodingInput, Document, ResourceName, useMedplum } from '@medplum/react';
+import { useState } from 'react';
+import { ValueSet } from '@medplum/fhirtypes';
+
+export function HomePage(): JSX.Element {
+  const medplum = useMedplum();
+
+  const [searchTerm, setSearchTerm] = useState('');
+  const [customValueSet, setCustomValueSet] = useState(`{
```

Comment:
One thing that might be confusing is that here you're using `expansion` to enumerate codes. This is necessary in this case because there isn't a code system for example.org/custome-allergies. 

However, for the existing `CodeSystems`, the proper way to do this would be to put these in the `ValueSet.compose.include`. 

To avoid confusing new users, I think it would be better for this placeholder example to use `ValueSet.compose.include` to take codes from an existing code system, eg. Snomed

Reply:
Switched this to use an RxNorm example